taskcluster/android_taskgraph/transforms/signing_apk.py

```diff
@@ -90,13 +75,7 @@ def set_signing_type(config, tasks):
 @transforms.add
 def set_index(config, tasks):
     for task in tasks:
-        index = {}
-        if (
-            config.params["tasks_for"] in ("cron", "github-release", "action")
```

Comment:
Question: IIUC, removing this if statement will add signing index to all tasks which includes `debug` on `git-push` for instance. I'm fine with that. I wonder if there's any other side effects. What does `taskgraph target-graph --diff` return?

Reply:
Thanks! This diff looks good to me. I don't see anything problematic with exposing these signed builds on the Taskcluster index.